source/features/useful-not-found-page.tsx

```diff
@@ -108,6 +109,15 @@ function init(): false | void {
 	}
 }
 
+async function initPRCommit(): Promise<void> {
+	const commitUrl = location.pathname.replace(/pull\/\d+\/commits/, 'commit');
```

Comment:
Feels pretty straight-forward since we already have `is404` in the same file

```suggestion
async function initPRCommit(): Promise<void | false> {
	const commitUrl = location.href.replace(/pull\/\d+\/commits/, 'commit');
	if (await is404(commitUrl)) {
		return false;
	}
```

How does this sound? Does it work?

Reply:
Nice!